plugins/welcome/index.js

```diff
@@ -0,0 +1,49 @@
+const BasePlugin = require('../base');
+const Commenter = require('../../commenter');
+
+class WelcomePlugin extends BasePlugin {
+  /**
+   * Whether this plugin processes edit actions
+   * @public
+   * @override
+   * @returns {Boolean} Whether this plugin processes edit actions
+   */
+  get processesEdits() {
+    return false;
+  }
+
```

Comment:
This is default, so you can omit it altogether.

Reply:
burned this